pkg/eks/tasks.go

```diff
@@ -431,3 +431,23 @@ func (c *ClusterProvider) appendCreateTasksForIAMServiceAccounts(cfg *api.Cluste
 		spec:            cfg,
 	})
 }
+
+// clusterSupportsDevicePlugin returns true if any of the node/managed node groups
+// uses an AMI family which requires the device plugin to be installed as
+// a daemonset
+func clusterSupportsDevicePlugin(cfg *api.ClusterConfig) bool {
```

Comment:
What do you think about naming this `clusterRequiresDevicePlugin`?

```suggestion
func clusterRequiresDevicePlugin(cfg *api.ClusterConfig) bool {
```